owl/prompts/summarization.py

```diff
@@ -1,5 +1,9 @@
 from ..core.config import Configuration
 
+def suggest_links_system_message(config: Configuration) -> str:
```

Comment:
Move this into own file under prompts/. It's not technically summarization. Maybe web_search.py or suggested_links.py?

Reply:
Done